### PR TITLE
`<TextButton/>`- Adding tiny styling for text, prefix & suffix and updating documentation

### PR DESCRIPTION
**Pull request by TalZa1810**
I need the styles in wix-ui-core in order to resolve this issue:
wix/wix-style-react#3239

### CODE REVIEW

File: packages/wix-ui-core/src/themes/backoffice/text-button/text-button.st.css [hunk 1]
```diff
@@ -408,3 +413,17 @@
   width: 18px;
   height: 18px;
 }
+
+.textButton.tiny .prefix{
+  padding-left: 6px;
+  margin: 0;
+  width: 15px;
```
Comment (TalZa1810) [inline packages/wix-ui-core/src/themes/backoffice/text-button/text-button.st.css]: TODO: change to 18px both height & width

File: packages/wix-ui-core/src/themes/backoffice/text-button/text-button.st.css [hunk 2]
```diff
@@ -379,6 +379,11 @@
   -st-mixin: sizeSmall;
 }
 
+.textButton.tiny{
+  height: 15px;
```
Comment (TalZa1810) [inline packages/wix-ui-core/src/themes/backoffice/text-button/text-button.st.css]: TODO: change to 18px both height

**Comment by TalZa1810** [comment]
@LittleTal Done

File: packages/wix-ui-core/src/themes/backoffice/text-button/text-button.st.css [hunk 3]
```diff
@@ -408,3 +413,17 @@
   width: 18px;
   height: 18px;
 }
+
+.textButton.tiny .prefix{
+  padding-left: 6px;
+  margin: 0;
+  width: 18px;
+  height: 18px;
```
Comment (NitayRabi) [inline packages/wix-ui-core/src/themes/backoffice/text-button/text-button.st.css]: `18px` vs. `15px`

File: packages/wix-ui-core/src/themes/backoffice/text-button/text-button.st.css [hunk 4]
```diff
@@ -408,3 +413,17 @@
...
+}
+
+.textButton.tiny .suffix{
+  padding-left: 6px;
+  margin: 0;
```
Comment (NitayRabi) [inline packages/wix-ui-core/src/themes/backoffice/text-button/text-button.st.css]: Is there a way to group margin and padding and similar props to a single class?

File: packages/wix-ui-core/stories/backoffice/button-next/showcase/text-Button/textButton-sizes.tsx [hunk 5]
```diff
@@ -40,7 +44,10 @@ export const TextButtonSizes = ({ style }: TextButtonSizesProps) => (
     code={example}
     description={description}
   >
-    <ButtonNext className={textButton('small')} prefixIcon={<ChevronDown />}>
+    <ButtonNext className={textButton('tiny')} prefixIcon={<ChevronDownSmall/>}>
```
Comment (NitayRabi) [inline packages/wix-ui-core/stories/backoffice/button-next/showcase/text-Button/textButton-sizes.tsx]: Do we have an enum to define `'tiny'/'small'` etc?

Comment (TalZa1810) [reply packages/wix-ui-core/stories/backoffice/button-next/showcase/text-Button/textButton-sizes.tsx]: No. I'll do it in a different PR.

**Pull request merged**